runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesConfig.java

```diff
@@ -43,4 +43,13 @@ default boolean enabled()
 	{
 		return true;
 	}
+	@ConfigItem(
+			keyName = "enableTooltips",
+			name = "Enable Tooltips",
+			description = "Configures whether or not to show tooltips"
```

Comment:
This should just be indented one tab. Not sure how the whole file got double indented.